client.go

```diff
@@ -71,14 +72,23 @@ func (c *Castle) Filter(ctx context.Context, req *Request) (RecommendedAction, e
 	if req.Context == nil {
 		return RecommendedActionNone, errors.New("request.Context cannot be nil")
 	}
-	r := &castleAPIRequest{
+	params := Params{
+		Email:    req.User.Email,
+		Username: req.User.Name,
```

Comment:
Is the name necessarily the username? I wonder if we're reusing the `name` field in our generic request type in a way that wasn't intended by the underlying API, or are they actually the same values?

Reply:
No so i actually was just thinking about this now, I remember us using either the account number or the email as username in a service somewhere but i can't think were.

Reply:
Not really sure whether to include this now, looking at the code from the castle-processor the username is seems to be either the email or account number depending on what the user used to sign in. The username value is contained in the request properties so I've just set that as the username, might be that the username and the email are duplicates in some cases but I don't think that will be an issue.